tests/test_matching_nnlo.py

```diff
@@ -77,3 +77,24 @@ def test_pegasus_sign():
     aS2 = A_singlet_2(N, sx, L)
 
     np.testing.assert_allclose(aS2[0, 0], ref_val, rtol=4e-5)
+
+def test_Bluemlein_Hg2():
+    # Test against Blumlein OME implementation.
+    # Only even moments are non zero
+    # This might be bacause they are computing only even moments at N3LO
+    # see https://arxiv.org/pdf/0904.3563.pdf (eq 8.50 and similar)
+    # Note there is a minus sign in the definition of L.
+
+    ref_val_l0 = [9.96091, 10.6616,9.27572, 8.25694, 7.49076]
+    ref_val_l10 = [289.097, 278.051, 223.261, 186.256, 160.027]
+    for n in range(1,6):
+        N = 2 * n
+        sx = get_sx(N)
+
+        L = 0.0
+        aS2 = A_singlet_2(N, sx, L)
+        np.testing.assert_allclose(aS2[2,0], ref_val_l0[n-1], rtol=6e-4)
```

Comment:
Mmm is `6e-4` really sufficient? ...

Reply:
Indeed this seems odd, since the other ones  I am trying with are matching to ``5e-6`` accuracy....
Unfortunately this Bluemlein expressions can not be used in python directly because they depends on this HarmonicSums package.
It's not fully clear to me why the computed only even moments also at NNLO, I need to check the reference...

Reply:
- Maybe you can bisect the expressions and check bit by bit? and hence verify your conjecture about `M[Li_3(1-x)](N)` of beeing potentially bugged?
- I think the reason  why they compute only even moments is related to the way the OPE works ... I faced the same problem for the NLO ad and one needs to be very careful about the correct analytically continuation into complex N

Reply:
Looking at Silvias expresions:
- the most complicated expression for `AQg123N` is `S[-2, 1, N]` - which we know via `g_3`: https://eko.readthedocs.io/en/develop/theory/Mellin.html#harmonic-sums
- so this should work at least on the same level as all the others (and as the `ad.NLO`)
- did you triple check NLO (since without this everything beyond is invisible)?
- if necessary you can translate this expressions using https://github.com/N3PDF/eko/wiki/Parse-NLO-expressions (or otherwise I can do it for you ...)

Reply:
Hi Felix, Okay I'll work on this. yes probably I can translate ``AQg123N`` it in python completely, but still it won't work because it has only the even moments, right? 
 no I haven't checked the NLO yet.

Reply:
I still believe in my comment I stated in the docs:
> where formally η=(-1)^N but in all singlet-like quantities it has to be analytically continued with 1 and with -1 elsewise.